src/lib/getWikipediaInfo.ts

```diff
@@ -0,0 +1,109 @@
+import { Result } from './scrape';
+import * as got from 'got';
+import * as createFuzzySet from 'fuzzyset.js';
+import { first } from 'lodash';
+
+const WIKIPEDIA_API_ENDPOINT = 'https://en.wikipedia.org/w/api.php';
+
+export type WikipediaData = {
+  url: string;
+  title: string;
+  thumbnail: {
+    source: string;
+    width: number;
+    height: number;
+  };
+};
+
+export async function getWikipediaInfo(
+  result: Result,
+  thumbnailHeight = 300,
+): Promise<Partial<WikipediaData>> {
+  const urlRequest = got(WIKIPEDIA_API_ENDPOINT, {
```

Comment:
It would've made it easier for me to understand this if it were called `pageRequest`